SharkCage/util/sign_sharkcage_release.bat

```diff
@@ -0,0 +1,26 @@
+echo off
+
+echo Use this batch script to sign the 'SharkCageApplication'.
+echo Make sure: 
+echo 1. You started the script with the 'Developer Command Prompt for VS 2017' 
+echo 2. You provided the path to your solution folder e.g.: C:\HTWG_shark_cage\SharkCage
+echo 3. You have the Yubikey with the certificate (including the private key) available.
+
+set path_to_solution=%1
+
+echo Now build the complete solution, when it is done press enter.
+pause >nul
+
+signtool sign /d "CageChooser" /tr http://timestamp.digicert.com /td sha256 /fd sha256 /sha1 ADBE74BD39789DD111815DE59C60D715143E4620 %path_to_solution%\CageChooser\obj\Release\CageChooser.exe
```

Comment:
The correct output path is: `%path_to_solution%\CageChooser\`__`bin`__`\Release\CageChooser.exe`
![image](https://user-images.githubusercontent.com/16246147/44310130-7454b180-a3d1-11e8-8624-31eb29e57784.png)

Not sure why the binary only copied to the `obj` folder.

Reply:
The binary is first copied to the `obj` folder and after that to the `bin`. That was the problem, why the `Chooser` and the `Configurator` wasn't singed on the last "preprerelease". The output in the bin folder is replaced by the content of the obj. Found after searching a round an issue for that but I'd have to look again to find the link...

Reply:
This was the link: https://stackoverflow.com/a/22567659/825532

Reply:
And found no way to configure that the installer should use the `bin` folder instead of the `obj`